packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -208,55 +208,6 @@ class ScrollButtons extends Component {
 		Spotlight.focus(isPrev ? this.prevButtonRef.current : this.nextButtonRef.current);
 	}
 
-	focusOnOppositeScrollButton = (ev, direction) => {
```

Comment:
Some UX requirements will be missing by removing this code.

Reply:
I intended to use `spotlightRule="grid"` option to move focus oppositeButton.
But I missed the code for normal scroller. I'll fix it now.